src/core.js

```diff
@@ -516,16 +511,18 @@ function extractStacktrace( e, offset ) {
 }
 
 function sourceFromStacktrace( offset ) {
-	var e = new Error();
-	if ( !e.stack ) {
+	var error = new Error();
+
+	// Ref https://github.com/jquery/qunit/issues/636
```

Comment:
good catch:

- Safari 7 (safari_previous)
- IE10
- IE11

If we drop support for Safari 6 (current is 8), the `.sourceURL` can also be dropped.